### ISSUE TITLE
DropDownMenuItem  external_link=True won't refresh the page

### ISSUE THREAD

**Issue by SteveGaoZhitao** [facultyai/dash-bootstrap-components#457]
Before opening:

- [Search for duplicate or closed issues](https://github.com/facultyai/dash-bootstrap-components/issues?utf8=%E2%9C%93&q=is%3Aissue)
- Read the [contributing guidelines](https://github.com/facultyai/dash-bootstrap-components/blob/master/.github/CONTRIBUTING.md)

Please fill out the below information as much as possible.

dash==1.13.4
dash-bootstrap-components==0.10.3
dash-core-components==1.10.1
dash-extensions==0.0.31
dash-html-components==1.0.3
dash-renderer==1.5.1
dash-table==4.8.1

### What is happening?

DropDownMenuItem  external_link=True isn't working

### What should be happening?

Expect the page will refresh if user click on the link, but it works like normal 

### Code

```python
# a file to test different navbar options
import pandas as pd 
import dash
import dash.dependencies as dd
from dash.dependencies import Input, Output, State
import dash_html_components as html
import dash_bootstrap_components as dbc

app=dash.Dash(__name__)

APP_TITLE='TestNav'

def get_navbar_dropdowns():
    # header of drop down
    dropdown_children=[
        dbc.DropdownMenuItem('n1',header=True),
        dbc.DropdownMenuItem('n1_1_abs_url',href='https://google.com'),
        dbc.DropdownMenuItem('n1_1_ext',href='google.com',external_link=True),  # not working
        dbc.DropdownMenuItem('extxxx',href='/xx',external_link=True),  # expecting refresh the page
        dbc.DropdownMenuItem('ext_target',href='google.com',external_link=True)
    ]
    # Make one dropdown item 
    nav_dropdowns=[
        dbc.DropdownMenu(
            children=dropdown_children,nav=True,in_navbar=True,label='lv1'
        )
    ]
    return nav_dropdowns

# navbar part 
nav_dropdowns=get_navbar_dropdowns()
nav=dbc.Nav(nav_dropdowns,className="ml-auto", navbar=True)
navbar = dbc.Navbar(
    children=dbc.Container(
        [
        html.A(
            dbc.Row(
                [
                    dbc.Col(
                        # dbc.NavbarBrand(APP_TITLE, className="_ngcontent-wpj-c5"),
                        dbc.NavbarBrand(APP_TITLE, className="ml-4"),

                        width={'size':3}
                        ),
                ],
                align="center",
                # align="left",
                no_gutters=True,
            ),
            href="/",
        ),
        dbc.Collapse(
                
                nav,
                id="navbar-collapse2",
                navbar=True,
            ),
    ],

    # for container,make the nav bar container to use full space
    fluid=True
    ),
    color="#c9170a",  # Set color of bar
    tag='nav',
    expand='lg'
)

app.layout=html.Div(
        children=[
        dbc.Row(
            [
                dbc.Col(navbar)
            ]
        ),
    ])

if __name__=='__main__':

    app.run_server(debug=True)
```

### Error messages

```bash
paste any error messages here
```

**Comment by tcbegley** [comment]
Hi @SteveGaoZhitao 

Thanks for reporting. Made a fix in #458

As a side note, this item

```python
dbc.DropdownMenuItem('n1_1_ext',href='google.com',external_link=True)
```

will not work even with the fix as it will be treated like a relative link and navigate to `http://localhost:8050/google.com` or similar. Relative external links should be used for when you want to access a route on the server such as a Flask endpoint, if you want to link to a different domain you'll have to specify an absolute url.

**Comment by SteveGaoZhitao** [comment]
Hi @tcbegley 
Thanks for this side note!

Do we have a  plan to get this fixed in the next version?
I can make this change locally but the apps in docker container must get packages with pip

**Comment by tcbegley** [comment]
Hey @SteveGaoZhitao 

Sorry for the delay, had a busy week but wanted to add some tests to make sure this doesn't happen again before merging. The fix is now available in version 0.10.7, so you should be able to simply update now 🙂

**Comment by SteveGaoZhitao** [comment]
Hi @tcbegley   Thanks a lot! It works now! 👍👍👍
